shopfloor/services/zone_picking.py

```diff
@@ -602,19 +601,15 @@ def _set_destination_location(self, move_line, quantity, confirmation, location)
         # if `confirmation is True
         # Ask confirmation to the user if the scanned location is not in the
         # expected ones but is valid (in picking type's default destination)
-        if not location.is_sublocation_of(
-            self.picking_type.default_location_dest_id
-        ) or not location.is_sublocation_of(
-            move_line.move_id.location_dest_id, func=all
-        ):
+        stock = self._actions_for("stock")
```

Comment:
seems you don't need to call `stock` anymore

Reply:
removed